src/Resources/config/services.yaml

```
@@ -38,7 +38,7 @@ services:
         port: '%env(MONSIEURBIZ_SEARCHPLUGIN_ES_PORT)%'
         elastically_mappings_directory: '%kernel.project_dir%/vendor/monsieurbiz/sylius-search-plugin/src/Resources/config/elasticsearch/mappings'
         elastically_index_class_mapping:
-          # @TODO Add it in config
+          documents-it_it: \MonsieurBiz\SyliusSearchPlugin\Model\Document\Result
```

```suggestion
          # @TODO Add it in config
          documents-it_it: \MonsieurBiz\SyliusSearchPlugin\Model\Document\Result
```

We need to keep the TODO